site/content/en/latest/performance/benchmark.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Benchmark
+---
+
+Envoy Gateway uses [nighthawk][nighthawk] for benchmarking, and mainly concerned with 
+its performance and scalability as a control-plane.
+
+The performance and scalability concerns come from several aspects for control-plane:
+
+- The consumption of memory and CPU.
+- The rate of configuration changes.
+
+## Run Benchmark Test
```

Comment:
this is more developer facing (should live in https://github.com/envoyproxy/gateway/blob/main/site/content/en/contributions/DEVELOP.md) than user facing

we should be publishing performance numbers here

Reply:
done